app/assets/stylesheets/simple_form/dropdown_select.scss

```diff
@@ -26,6 +26,14 @@
   }
 }
 
+.drop_rich_head,
```

Comment:
Selector `drop_rich_head` should be written in lowercase with hyphens<br>Selector `drop_rich_subhead` should be written in lowercase with hyphens